server/pa.js

```diff
@@ -16,7 +47,7 @@ const validateCreateSensorLocation = Validator.body([
   'particleDeviceID',
   'twilioNumber',
   'clientID',
-  'clickupToken',
+  'idToken',
```

Comment:
`idToken` sounds very generic to me. Up to you, but is there a way that you can make this name more descriptive?

Reply:
Fixed as per changes in brave-alert-lib! Anything that originated from google now has the 'google' prefix; e.g., googleIdToken, googleAccessToken, googleAuthCode.